tools/bridge/end2end-tests/tests/test_transfer.py

```diff
@@ -117,6 +126,9 @@ def start(self):
             self.terminate()
             raise
 
+    def is_started(self):
+        return self.process is None
```

Comment:
Now you added it but do not even use it yourself?
Unuesed code can easily lead to broken code, which is also what happened, it should be
`self.process is not None`

If you dont use it, maybe better remove it.